Modules/_testcapimodule.c

```diff
@@ -1016,6 +1016,31 @@ test_buildvalue_N(PyObject *self, PyObject *Py_UNUSED(ignored))
     Py_RETURN_NONE;
 }
 
+static PyObject *
+test_buildvalue_ignore_char(PyObject *self, PyObject *Py_UNUSED(ignored))
+{
+    PyObject *res = Py_BuildValue("(i )", 0);
+    if (res == NULL) {
+        return NULL;
+    }
+    Py_DECREF(res);
+    res = Py_BuildValue("i, ", 0);
+    if (res == NULL) {
+        return NULL;
+    }
+    Py_DECREF(res);
+    res = Py_BuildValue("i, :", 0);
+    if (res == NULL) {
+        return NULL;
+    }
+    Py_DECREF(res);
+    res = Py_BuildValue("i, :   ", 0);
```

Comment:
Why so many tests? Would not be enough one test which contains all ignored characters? Also `\t` is not covered.

Reply:
Okay.

Reply:
IMO, consider simply exposing `Py_BuildValue` via the `_testcapi` module, and write the tests in Python.

Reply:
It is not easy to expose a C function which takes variable number of arguments of arbitrary types.